stack/app/api/v1/stream.py

```diff
@@ -0,0 +1,205 @@
+import functools
```

Comment:
Moved this file from /utils to appropriate location and modified to use state registry